rllib/utils/nested_dict.py

```diff
@@ -124,7 +124,7 @@ def __init__(
     ):
         # shallow dict
         self._data = dict()  # type: Dict[str, Union[T, NestedDict[T]]]
-        x = x or {}
+        x = x if x is not None else {}
```

Comment:
This is a curious issue where some values != None can not be represented as boolean values, for example arrays with more than one element.